ui/src/app/pages/workspace/workspace-wrapper/component.ts

```diff
@@ -58,6 +58,12 @@ export class WorkspaceWrapperComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit(): void {
+    if (!!localStorage.getItem('sidebarState')) {
```

Comment:
nit: pull out the `sidebarState` value into a constant. I think we're (loosely) following the convention of using capital letters for local storage keys. 

Also, prefixing the key value with `WORKSPACE` may be helpful as we're planning to add the side bar component to the notebooks page.

Reply:
Okay, now I see this convention used in the `signed-in` component also. Fixed.